docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+id: macos-opentelemetry
+title: macos - OpenTelemetry Collector
+sidebar_label: macOS - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for MacOS.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for macOS allows you to monitor the performance and resource utilization of hosts and processes that your mission critical applications are dependent upon. The app consists of predefined searches and dashboards that provide visibility into your environment for real-time or historical analysis.
+
+Preconfigured dashboards provide insight into CPU load, memory, network, file descriptors, page faults, and TCP connectors. This app uses OpenTelemetry, an open-source collector for metrics.
+
+We use the OpenTelemetry collector for metrics collection.
+
+OpenTelemetry collector runs on the macOS machine, and uses the [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver) to obtain host and process metrics, and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-Schematics.png')} alt="Schematics" />
+
+## Fields Created in Sumo Logic for macOS
+
+Following are the tags which will be created as part of macOS App install, if not already present. 
+
+**Across apps**
+
+- **`sumo.datasource`** - Has fixed value of **mac**.
+
+## Collecting Logs, Metrics & installing macOS app
+
+Here are the steps for collecting metrics and installing the app.
+
+### Step 1: Set up Collector
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.  
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for macOS Collection.
+
+There are no parameters required for the macOS app to install. One can add a custom tag if required in this step.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-YAML.png')} alt="YAML" />
+
+:::note
```

Comment:
This whole note section needs to be removed

Reply:
All of Step 2?